dolphinscheduler-ui/src/js/conf/home/pages/user/pages/account/_source/info.vue

```diff
@@ -110,6 +111,7 @@
           phone: param.phone
         })
         this.createUserDialog = false
+        this.getUserInfo()
```

Comment:
Good job!

Maybe it's better to replace the following lines
```
this.createUserDialog = false
this.getUserInfo()
```
with
```
this.getUserInfo().finally(() => {
  this.createUserDialog = false
})
```

Reply:
@chengshiwen  Thanks for your suggestion, I think what you said makes sense. solve it

Reply:
@kyoty Please resolve the code style introduced by spaces, thanks~